Doc/extending/newtypes.rst

```diff
@@ -244,7 +236,7 @@ doesn't do anything. It can't even be subclassed.
 Adding data and methods to the Basic example
 --------------------------------------------
 
-Let's expend the basic example to add some data and methods.  Let's also make
+Let's extend the basic example to add some data and methods.  Let's also make
```

Comment:
"extend"
https://docs.python.org/3/extending/newtypes.html#adding-data-and-methods-to-the-basic-example